testbed/package-lock.json

```diff
@@ -1953,6 +1953,711 @@
             "aws-sdk": "^2.848.0",
             "glob": "^7.1.7",
             "yargs": "^16.2.0"
+          },
```

Comment:
Were we missing these? Don't see any code change that introduces these.

Reply:
Yep. It's considered good practice to check in package-lock, similar to go.sum